docs/modules/nifi/pages/troubleshooting/index.adoc

```diff
@@ -21,6 +21,29 @@ spec:
 
 == `HTTP ERROR 400 Invalid SNI`
 
+=== Local PORT-FORWARD
+
+Since NiFi requires a valid SNI (Server Name Indication), we need to configure the nifi property
+[source,yaml]
```

Comment:
```suggestion
Since NiFi requires a valid SNI (Server Name Indication), for local testing you need to configure the NiFi property:
[source,yaml]
```